bolt-sidecar/src/config.rs

```diff
@@ -82,6 +86,7 @@ impl Default for Config {
             private_key: Some(random_bls_secret()),
             mevboost_proxy_port: 18551,
             limits: Limits::default(),
+            validator_indexes: vec![0], // Default to 0 index
```

Comment:
I think it makes more sense to initialize this to an empty array 

```suggestion
            validator_indexes: Vec::new()
```

Reply:
ok! then we should also make the `validator_indexes` argument as **required**?